connectors/sources/google_drive.py

```diff
@@ -246,8 +502,27 @@ def _google_drive_client(self):
         if "universe_domain" in json_credentials:
             json_credentials.pop("universe_domain")
 
-        return GoogleDriveClient(
+        return GoogleDriveClient(json_credentials=json_credentials)
+
+    @cached_property
+    def google_admin_directory_client(self):
+        """Initialize and return the GoogleAdminDirectoryClient
+
+        Returns:
+            GoogleAdminDirectoryClient: An instance of the GoogleAdminDirectoryClient.
+        """
+        self._validate_service_account_json()
+
+        json_credentials = json.loads(self.configuration["service_account_credentials"])
+
+        # Google Service Account JSON includes "universe_domain" key. That argument is not
+        # supported in aiogoogle library, therefore we are skipping it from the credentials payload
+        if "universe_domain" in json_credentials:
+            json_credentials.pop("universe_domain")
```

Comment:
```suggestion
        remove_universe_domain(json_credentials)
```

nit: Could be extracted to a method to use it for the `GoogleAdminDirectoryClient` and the `GoogleDriveClient`